### ISSUE TITLE
Function and Css

### ISSUE THREAD

**Issue by Ahamza24** [Ahamza24/I-GoSure#1]
The first issue I have is the function some of them are not being called I have checked multiple times but couldn't see why.

the next issue is with CSS for some reason I can't fully adjust the button position or the footer on some pages like the contactus.html and quotes.html. pls and also something with nav and position of content when I change to mobile view.

**Comment by eric-foster** [comment]
In regards to your first issue, what all seems to not be calling?

**Comment by Ahamza24** [comment]
> In regards to your first issue, what all seems to not be calling?

the function I have on main.js

**Comment by eric-foster** [comment]
Any chance you know which functions are not working for you? I have the file open so i can take a look directly and do some debugging.

**Comment by Ahamza24** [comment]
> Any chance you know which functions are not working for you? I have the file open so i can take a look directly and do some debugging.

GetRegteration and handleaccept payment are not working

**Comment by eric-foster** [comment]
![Screenshot (65)](https://github.com/Ahamza24/I-GoSure/assets/109866223/111b6c6b-0870-4ad6-80b3-36f376da97af)

I have managed to narrow down the errors to this. but something is blocking the file overall. I think it has something to do with this but I am unsure.

Uncaught TypeError: Failed to resolve module specifier "firebase/app". Relative references must start with either "/", "./", or "../".

**Comment by Ahamza24** [comment]
> ![Screenshot (65)](https://private-user-images.githubusercontent.com/109866223/294798502-111b6c6b-0870-4ad6-80b3-36f376da97af.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ2Nzk1MjUsIm5iZiI6MTcwNDY3OTIyNSwicGF0aCI6Ii8xMDk4NjYyMjMvMjk0Nzk4NTAyLTExMWI2YzZiLTA4NzAtNGFkNi04MGIzLTM2ZjM3NmRhOTdhZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEwOFQwMjAwMjVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yZmIxNGYzM2MxMDQ3ZjI3NTVlNGM1YTMzZTBlMGVkMzFmOTE3MmY0NWVkYzk3ZjkzYmJiZjUyNDBhZjFjZWUwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.zwki1ZtrywQAS_am_JGR-rVL7GuyciEvm-Q3FP_lE-g)
> 
> I have managed to narrow down the errors to this. but something is blocking the file overall. I think it has something to do with this but I am unsure.
> 
> Uncaught TypeError: Failed to resolve module specifier "firebase/app". Relative references must start with either "/", "./", or "../".

Ok, what about the function do you if I resolved it will run fully

**Comment by eric-foster** [comment]
I couldn't find a solution to that yet. I did take notice that no matter the function you try to call there, it gives the same error. I tried this code at the top of the file:

function test() {
  console.log('test')
}

So my guess is there is something that isn't connecting between the two files.

I added the same code block above in a script file just above the <main> tag to test this theory and it worked as expected. So it has to be related to a disconnect somewhere between the quote.html file and main.js file.

**Comment by eric-foster** [comment]
here is the test case:
![Screenshot (66)](https://github.com/Ahamza24/I-GoSure/assets/109866223/4903a3a4-2aca-42cf-9347-002b5a6aa139)

**Comment by Ahamza24** [comment]
> here is the test case: ![Screenshot (66)](https://private-user-images.githubusercontent.com/109866223/294800487-4903a3a4-2aca-42cf-9347-002b5a6aa139.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ2ODA0MTUsIm5iZiI6MTcwNDY4MDExNSwicGF0aCI6Ii8xMDk4NjYyMjMvMjk0ODAwNDg3LTQ5MDNhM2E0LTJhY2EtNDJjZi05MzQ3LTAwMmI1YTZhYTEzOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEwOFQwMjE1MTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01ZDZhMWY3ODllMjQ3MDdmYzFiNzVhNjNkNTQ5MDhkNjE0ZjRiZjQyNWFmNGIwYTU0YzM4Y2FhMTQ4NTE2ZjM1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.7aZbgaszbrRFrmdVTGzDNjPDLLN5Uq7JVPHB4T6OqOE)

Ok think I might just reloading the back from start again. Thanks this has given some ideas to try.

**Comment by eric-foster** [comment]
Of course. the only part I also noticed is that I can tell its connecting somehow because its catching that you have a variable 'admin' declared several times (not sure why as having them declared multiple times doesn't do anything for the code in terms of where they are currently placed) but ya. I saw that and was confused as to why the functions are working.

**Comment by Ahamza24** [comment]
> Of course. the only part I also noticed is that I can tell its connecting somehow because its catching that you have a variable 'admin' declared several times (not sure why as having them declared multiple times doesn't do anything for the code in terms of where they are currently placed) but ya. I saw that and was confused as to why the functions are working.

The admin is just something I have their for authentication system in the future if need, it came the one or 2 of the function I have there already. But that has been a massive, I started the project a week ago but only now I have some clarity. Thanks

**Comment by eric-foster** [comment]
> > Of course. the only part I also noticed is that I can tell its connecting somehow because its catching that you have a variable 'admin' declared several times (not sure why as having them declared multiple times doesn't do anything for the code in terms of where they are currently placed) but ya. I saw that and was confused as to why the functions are working.
> 
> The admin is just something I have their for authentication system in the future if need, it came the one or 2 of the function I have there already. But that has been a massive, I started the project a week ago but only now I have some clarity. Thanks

Got it. Good luck to you. hopefully you can find a solution.
